google/resource_storage_bucket_object.go

```diff
@@ -108,6 +112,21 @@ func objectGetId(object *storage.Object) string {
 	return object.Bucket + "-" + object.Name
 }
 
+func createSourceState(src interface{}) string {
+
```

Comment:
Minor quibble: this space can be removed

Reply:
Done